STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerInstanceTest.testFindAndSetAttributesSetInstanceGeneric
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerInstanceTest.java
CLEANER: de.strullerbaumann.visualee.examiner.cdi.TestGroup100Case0.test270
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 22
NEW CLEANER SIZE: 2
ELAPSED TIME: 76.84300017356873
==========================
@@ -101,0 +101,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
